vxquery-xtest/src/test/resources/VXQueryCatalog.xml

```diff
@@ -39,6 +39,8 @@
 <!ENTITY SingleQuery SYSTEM "cat/SingleQuery.xml">
 <!ENTITY SingleAlternateQuery SYSTEM "cat/SingleAlternateQuery.xml">
 
+<!ENTITY HDFSAggregatePartition1Queries SYSTEM "cat/HDFSAggregatePartition1Queries.xml">
```

Comment:
You re right,I removed partition from names and left it as aggregate tests for hdfs that executes collection reads.